.sonarcloud.properties

```diff
@@ -9,6 +9,7 @@ sonar.exclusions=prime-router/src/test/**, prime-router/src/main/kotlin/cli/test
 # sonar.tests=
 # sonar.test.exclusions=
 # sonar.test.inclusions=
+sonar.javascript.lcov.reportPaths=frontend-react/coverage/lcov.info
```

Comment:
sonarcloud [configuration](https://docs.sonarcloud.io/advanced-setup/automatic-analysis/#additional-analysis-configuration) appears to be much more limited than sonarqube.  

Let me know if you find that this works.